src/app/event-form/event-form.component.html

```diff
@@ -29,6 +29,7 @@
               name="location" 
               placeholder="Enter Event Location"
               required #location="ngModel">
+            <div [hidden]="location.pristine || !!location.value" class="alert alert-danger">Location is required</div>
```

Comment:
We have alert messages for missing fields for name, description and location. Do we want to include alert messages for the other fields (time, date, etc)? Family friendly checkbox could probably be left as is.

Reply:
Good catch - I didn't even notice that :)

Reply:
Changes made!